core-bundle/src/Resources/contao/classes/Backend.php

```diff
@@ -909,7 +909,7 @@ public static function addPageIcon($row, $label, DataContainer $dc=null, $imageA
 		$label = '<a href="' . \Backend::addToUrl('pn=' . $row['id']) . '" title="' . \StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['selectNode']) . '">' . $label . '</a>';
 
 		// Return the image
-		return '<a href="contao/main.php?do=feRedirect&amp;page=' . $row['id'] . '" title="' . \StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['view']) . '"' . (($dc->table != 'tl_page') ? ' class="tl_gray"' : '') . ' target="_blank">' . \Image::getHtml($image, '', $imageAttribute) . '</a> ' . $label;
+		return '<a href="contao/main.php?do=feRedirect&amp;page=' . $row['id'] . '" title="' . \StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['view']) . '"' . (($dc->table != 'tl_page') ? ' class="tl_gray"' : '') . ' target="_blank" rel="noreferrer noopener">' . \Image::getHtml($image, '', $imageAttribute) . '</a> ' . $label;
```

Comment:
I am not sure if adding `noreferrer noopener` here will break the front end preview. @contao/developers WDYT?

Reply:
Internal links (like the FE preview) don’t need `noreferrer noopener` IMO.